app/index.html

```diff
@@ -12,6 +12,7 @@
         <!-- bower:css -->
         <link rel="stylesheet" href="../bower_components/jquery.layout/dist/layout-default-latest.css" />
         <link rel="stylesheet" href="../bower_components/videojs/dist/video-js/video-js.css" />
+        <link rel="stylesheet" href="../bower_components/jquery-notific8/dist/jquery.notific8.min.css" />
```

Comment:
Yeah that could be done although I didn't see the need for it because we won't be changing anything in notific8. For debugging the source is any ways downloaded on the developers machine in the bower package, so it's one and the same thing in my opinion. If not, its a small change anyways, so I can put that in.